spec/models/framework/miso_fields_spec.rb

```diff
@@ -0,0 +1,55 @@
+require 'rails_helper'
+
+RSpec.describe Framework::MisoFields do
+  let(:framework_short_name) { 'RM3787' }
+
+  let(:miso_fields) { Framework::MisoFields.new(framework_short_name) }
+
+  describe '#filename' do
+    subject { miso_fields.filename }
+
+    it { is_expected.to eql(Rails.root.join('data', 'framework_miso_fields.csv')) }
+  end
+
+  describe 'operating on a single #framework_short_name' do
```

Comment:
I'm not sure I follow the wording for this describe block.

Reply:
Fair. Will reword.

Reply:
Has disappeared now, there was – as you point out elsewhere – no need to have this public.